pom.xml

```diff
@@ -164,6 +164,7 @@
                             <systemPropertyVariables>
                                 <profile.id>${profile.id}</profile.id>
                                 <postgresql.13.image>docker.io/library/postgres:13.6</postgresql.13.image>
+                                <postgresql.latest.image>docker.io/library/postgres:15.4</postgresql.latest.image>
```

Comment:
Move the other examples to use one postgresql, ideally 15.4 based

Reply:
done